src/static/css/module/index.less

```diff
@@ -37,43 +37,14 @@ fieldset.submit_btn > p {
   HERO
   ---------------------------- */
 
-.hero_wrapper {
-  margin: 0 auto;
-  position: relative;
-  clear: both;
-  max-width: @grid_wrapper-width;
-}
-
-#hero-bg-image {
-  position: absolute;
-  width: 100%;
-  z-index: -1;
-  
-  .respond-to-min(720px, {
-    background: @neutral-20 url('http://www.consumerfinance.gov/owning-a-home/static/img/oah-hero-image.png');
-    background-repeat: no-repeat;
-    background-size: auto 230px;
-    background-position: center top;      
-    background-size: auto 240px;
-    height: 240px;
-  });
-}
-
-.content_hero {
-  // background image on the content element for small screens only, so we can have a flexible height
-  background: @neutral-20 url('http://www.consumerfinance.gov/owning-a-home/static/img/oah-hero-image.png');
-  background-position: 0 0;
-
-  .respond-to-min(720px, {
-    padding-top: 0;
-    padding-bottom: 0;
-    height: 240px;
-    min-height: 240px;
-    background-color: transparent;
+.content-hero {
```

Comment:
instead of creating a new class I would recommend keeping the previous name and just applying your changed styles to that (assuming it is not used elsewhere)